src/config/swagger/tags.ts

```diff
@@ -1,4 +1,5 @@
 export const SWAGGER_TAGS = {
   HEALTH_CHECK: 'Health Check',
   AUTH: 'Auth',
+  FEE_CHALLAN: 'Fee Challan',
```

Comment:
Renaming will be reflected here as well.

Reply:
Noted